build.sh

```diff
@@ -1,24 +1,12 @@
 #!/usr/bin/env bash
```

Comment:
do we need to keep this build file? I would rather move the versions to Dockerfile, as in https://github.com/znly/docker-protobuf/blob/d22b5e13e718e6378635858e04bd8a6395984dfc/Dockerfile#L4